draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -43,75 +43,37 @@ organization="Ping Identity"
 
 .# Abstract
 
-This specification defines a mechanism for selective disclosure of individual elements of a JSON object
-used as the payload of a JSON Web Signature (JWS) structure.
-It can be used for multiple applications, including but not limited to the selective disclosure of JSON Web Token (JWT) claims.
+This specification defines a mechanism for the selective disclosure of individual elements of a JSON-encoded data structure used as the payload of a JSON Web Signature (JWS).
+The primary use case is the selective disclosure of JSON Web Token (JWT) claims.
 
 {mainmatter}
 
 # Introduction {#Introduction}
 
-This document specifies conventions for creating JSON Web Signature (JWS) [@!RFC7515]
-structures with JSON [@!RFC8259] objects as the payload while supporting selective disclosure of individual elements of that JSON.
-Because JSON Web Token (JWT) [@!RFC7519] is a very prevalent application of JWS with a JSON payload, the selective disclosure of JWT claims receives primary treatment herein. However, that does not preclude the mechanism's applicability to other applications of JWS with JSON payloads.
-
-The JSON-based representation of claims in a signed JWT is
-secured against modification using JWS digital
-signatures. A consumer of a signed JWT that has validated the
-signature can safely assume that the contents of the token have not been
-modified.  However, anyone receiving an unencrypted JWT can read all the
-claims. Likewise, anyone with the decryption key receiving encrypted JWT
-can also read all the claims.
-
-One of the common use cases of a signed JWT is representing a user's
-identity. As long as the signed JWT is one-time
-use, it typically only contains those claims the user has consented to
-disclose to a specific Verifier. However, there is an increasing number
-of use cases where a signed JWT is created once and then used a number
-of times by the user (the "Holder" of the JWT). In such use cases, the signed JWT needs
-to contain the superset of all claims the user of the
-signed JWT might want to disclose to Verifiers at some point. The
-ability to selectively disclose a subset of these claims depending on
-the Verifier becomes crucial to ensure minimum disclosure and prevent
-Verifiers from obtaining claims irrelevant for the transaction at hand.
-SD-JWTs defined in this document enable such selective disclosure of JWT claims.
-
-One example of a multi-use JWT is an Issuer-signed
-credential that contains the claims about a subject, and whose authenticity can be
-cryptographically verified.
-
-Similar to the JWT specification on which it builds, this document is a product of the
```

Comment:
I dropped this part. If we want to keep it, I suggest including it under a separate "## Scope" headline below the intro or below the feature list.

Reply:
I wrote some of that in response to questions/comments by Hannes about the appropriateness of this work in the OAUTH WG. Maybe the relationship to OAuth and JWT stuff isn't needed anymore. And the generally applicable bit is pretty much implied, in my opinion, even if Dick didn't see it.